### PR TITLE
add Gogland ide prefs

### PR DESCRIPTION
**Pull request by professor**
includes templates for ginkgo

Signed-off-by: Todd Sedano <professor@gmail.com>

### CODE REVIEW

File: cli/lib/cli/ide/gogland_user_pref_dir.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+module Cli
+  module Ide
+    class GoglandUserPrefDir < JetBrainsIdeUserPrefDir
+      def ide_pref_dir_name_without_version
+        "Gogland"
+      end
+
+      def default_ide_pref_dir_version
+        "whatever"
```
Comment (professor) [inline cli/lib/cli/ide/gogland_user_pref_dir.rb]: @moonmaster9000 

We noticed that the value of default_ide_pref_dir_version did not matter. We ran the ruby code and it detected 1.0 automatically. In the past, I've set this explicitly to the IDE_VERSION. It's been unclear how to handle this value when the IDE changes version numbers. For us, "whatever" more clearly aligns with the behavior we are seeing from the code. Any thoughts?

**Comment by moonmaster9000** [comment]
If there's no Gogland IDE currently installed (I.e., there's no prefs for a
Gogland IDE already in ~/Library/Preferences), then with the configuration
you've provided, running pivotal_ide_prefs would setup Gogland prefs in
~/Library/Preferences/Goglandwhatever. In other words, the
"default_ide_pref_dir_version" is there for the scenario where you're
running the pref installed *before* you've installed the IDE. The need for
that ability came from the desire to run pivotal_ide_prefs via tools like
sprout wrap before a human had ever opened the IDE app.

**Comment by professor** [comment]
@moonmaster9000, I fixed the default_ide_pref_dir_version. Thanks for explaining the change. At some point I'd love to catch up with you about the Labs Practice council. Todd

**Comment by professor** [comment]
@moonmaster9000, we'd like to roll this out to the other machines on my team. Ok to merge?

**Comment by moonmaster9000** [comment]
Merged! Apologies for the delay, thank you for pinging me.

**Pull request merged**